includes/bookmark.inc

```diff
@@ -841,6 +841,10 @@ abstract class Bookmark implements BookmarkInterface {
         $params[] = $form_state['values']['bookmarks']['fieldset']['format_opts']['export_styles'][$key];
```

Comment:
Hi, there is a danger. The export function that is called could not know that export_styles was not present (means line 841 was not executed) and could interpret $form_state as being that inside $params, because it would ocupy the second position (looking at key "1") in the array. That is the reason also i find $params without associative keys a bit complex